### DISCUSSION TITLE
vscode extension doesn't work on wsl mode

### DISCUSSION THREAD

**Discussion by henriquecustodia** [massCodeIO/massCode#102] [Q&A]
I've opened an issue [vscode-extension](https://github.com/massCodeIO/assistant-vscode/issues/4).  

Has anyone discovered a workaround for it?

**Comment by sangyuxiaowu** [comment]
The extension of vscode listens to localhost, and the localhost in wsl is just itself.

You can try to install masscode to wsl too.

![微信截图_20220724170328](https://user-images.githubusercontent.com/17669599/180640110-45b26cfe-b524-4063-88f1-86a14ca5d0dc.png)
